AppCenterData/AppCenterData/Model/MSDocumentChange.m

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSDocumentChange.h"
+
+@implementation MSDocumentChange
+
+static NSString *const kMSDocumentChangeId = @"id";
+static NSString *const kMSDocumentChangePartition = @"ptn";
+static NSString *const kMSDocumentChangeOperation = @"op";
+static NSString *const kMSDocumentChangeTimestamp = @"_ts";
```

Comment:
In the design review discussions we abbreviated the properties to reduce the message size since they are repeated for every document included in the change notification and the more overhead means the smaller the batch sizes we can support. The underscore for timestamp is because this is mapped directly from the document in cosmosdb and this is the format for timestamp there.

Yes partition is exactly that - a value to differentiate user and public/readonly data. In the future this will also support the name of shared partitions